go/coordinator/internal/metastore/db/dao/record_log.go

```diff
@@ -1,9 +1,46 @@
 package dao
 
 import (
+	"github.com/chroma/chroma-coordinator/internal/metastore/db/dbmodel"
+	"github.com/chroma/chroma-coordinator/internal/types"
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
 	"gorm.io/gorm"
+	"time"
 )
 
 type recordLogDb struct {
 	db *gorm.DB
 }
+
+func (s *recordLogDb) PushLogs(collectionID types.UniqueID, recordsContent [][]byte) (int, error) {
+	err := s.db.Transaction(func(tx *gorm.DB) error {
+		var timestamp = time.Now().UnixNano()
+		var collectionIDStr = types.FromUniqueID(collectionID)
+		log.Info("PushLogs",
+			zap.String("collectionID", *collectionIDStr),
+			zap.Int64("timestamp", timestamp),
+			zap.Int("count", len(recordsContent)))
+
+		var recordLogs []*dbmodel.RecordLog
+		for index := range recordsContent {
+			recordLogs = append(recordLogs, &dbmodel.RecordLog{
+				CollectionID: collectionIDStr,
+				Timestamp:    timestamp,
+				Record:       &recordsContent[index],
+			})
+		}
+		err := tx.CreateInBatches(recordLogs, len(recordLogs)).Error
```

Comment:
Just to confirm, this will commit if the transaction is successful.

Reply:
test won't pass otherwise